lib/websockets.py

```diff
@@ -69,22 +72,27 @@ def make_request(self, request_id):
         d = json.loads(s)
         addr = d.get('address')
         amount = d.get('amount')
-        return addr, amount
+        scripthash = bitcoin.address_to_scripthash(addr)
+        hash2address[scripthash] = address
+        return scripthash, amount
```

Comment:
`bitcoin` is not imported
instead of `hash2address`, you want `self.hash2address`
`addr` instead of `address`